resources/views/details-box-mobile.blade.php

```diff
@@ -1,8 +1,8 @@
 <div class="flex justify-between">
     <div class="{{ $titleClass ?? 'w-32' }}">{{ $title }}</div>
 
-    <div class="flex items-center space-x-3">
-        <div>{{ $slot }}</div>
+    <div class="flex items-center space-x-3 {{ $wrapperClass ?? false }}">
+        <div class="{{ $slotClass ?? 'false' }}">{{ $slot }}</div>
```

Comment:
is this needed or could it be passed a `div` with the proper classes as part of the slot instead?

Reply:
Was needed for my usecase, need to apply on both the slot & the parent div

Reply:
https://github.com/ArkEcosystem/nodem/pull/245/files#diff-6d86f9014b05fcb82fee7fcb48e7770d0343e323886a08c71de75f418477589dR53